### ISSUE TITLE
GB vs UK Country code

### ISSUE THREAD

**Issue by nicholasshirley** [eskerda/pybikes#270]
I'm not sure if this is the right place to raise this, but I was flipping through the data and I noticed that there is one (and only one) bike share company that uses the country code "UK" instead of "GB" as the rest of the United Kingdom companies do. Here is the entry:

```
  {
      "company": [
        "Groundwork",
        "Slough Borough Council",
        "ITS"
      ],
      "href": "/v2/networks/cycle-hire-slough",
      "id": "cycle-hire-slough",
      "location": {
        "city": "Slough",
        "country": "UK",
        "latitude": 51.51135,
        "longitude": -0.591562
      },
      "name": "Cycle Hire"
    },
```

**Comment by nicholasshirley** [comment]
I see something similar is already brought up on #89 though this station is not included in the list.

**Comment by nicholasshirley** [comment]
Fixed with #271
